src/TestsConfig.cpp

```diff
@@ -0,0 +1,24 @@
+#include"TestsConfig.h"
+
+std::string test_config(const std::string cname)
```

Comment:
Good. But see how we did it you dont need to use it as you used. 
For future tests Just create new folder called tests (same level as src) and place it there not in src dir

Reply:
it's not a test it's config for the leakage tests.
I didn't want to use  boost for it